apps/src/aiTutor/levelEditors/AITutorFields.tsx

```diff
@@ -0,0 +1,75 @@
+import React, {useState} from 'react';
+import moduleStyles from '@cdo/apps/lab2/levelEditors/aichatSettings/edit-aichat-settings.module.scss';
+import Checkbox from '@cdo/apps/componentLibrary/checkbox/Checkbox';
+import {generalChatSystemPrompt} from '../constants';
+
+interface AITutorFieldsProps {
+  isAvailable: boolean;
+  levelbuilderPrompt: string;
+}
+
+const AITutorFields: React.FunctionComponent<AITutorFieldsProps> = ({
+  isAvailable,
+  levelbuilderPrompt,
+}) => {
+  const [aiTutorAvailable, setAiTutorAvailable] =
+    useState<boolean>(isAvailable);
+  const [levelSpecificPrompt, setlevelSpecificPrompt] =
+    useState<string>(levelbuilderPrompt);
+
+  return (
+    <div>
+      <input
+        type="hidden"
+        id="level_ai_tutor_available"
+        name="level[ai_tutor_available]"
+        value={aiTutorAvailable.toString()}
+      />
+
+      <div className={moduleStyles.fieldSection}>
+        <div className={moduleStyles.fieldRow}>
+          <label
+            htmlFor="aiTutorAvailableCheckbox"
+            className={moduleStyles.inlineLabel}
+          >
+            Enable AI Tutor on Level
+          </label>
+          <Checkbox
+            name="aiTutorAvailableCheckbox"
+            checked={aiTutorAvailable || false}
+            onChange={e => {
+              setAiTutorAvailable(!aiTutorAvailable);
+            }}
+          />
+        </div>
+        <br />
+
+        <input
+          type="hidden"
+          id="level_ai_tutor_level_specific_prompt"
+          name="level[ai_tutor_level_specific_prompt]"
+          value={levelSpecificPrompt.toString()}
```

Comment:
do we need `toString` here? The value is already a string, right?

Reply:
When I tested it without `toString` it didn't work and when I added it, then it worked. I can't say I understand why at this point.

Reply:
I wonder if the issue was that you needed `toString` on `aiTutorAvailable` because it's a boolean, but here you shouldn't need it. I tested locally and removed `toString` here and it seemed to work.

Reply:
Thanks! Made this update.